mods/cnc/maps/nod10b/rules.yaml

```diff
@@ -14,7 +14,7 @@ World:
 			easy: Easy
 			normal: Normal
 			hard: Hard
-		Default: easy
+		Default: normal
 		Locked: false
```

Comment:
`Locked: false` is the default, so now that the `campaign-maprules` override is removed these can all go too.